UFOMG/ufomg-fe
# Testing/cypress integration tests

### Description

- This branch implements basic integration tests using Cypress
- These tests will need to be refactored and potentially added to as the application changes

### Closes issue(s)

Closes #18 

### Testing

- Run `npx cypress open`
- Select 'Run 1 integration spec'
- See all passing tests


### Changes include

### Checklist
- [X] I have written tests for this code (happy & sad)
- [X] I ran full test suite - all tests passing 
- [ ] Lint code

### Other comments
